tldr/hybrid_extractor.py

```
@@ -449,19 +469,19 @@ def _safe_parse(self, parser: Any, source: bytes, file_path: Path, language: str
         try:
             return parser.parse(source)
         except Exception as e:
-            logger.error(f"Tree-sitter parse failed for {file_path} ({language}): {e}")
+            logger.error("Tree-sitter parse failed for %s (%s): %s", file_path, language, e)
```

Ruff (0.14.14)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)